### DISCUSSION TITLE
Why has been loaded, the hash value can't see

### DISCUSSION THREAD

**Discussion by computer817** [Chia-Network/chia-blockchain#6879] [Support]
0x72a64aa64d308cb14556727b2109fbf9a2a421d8144e4872f2e4c3ba2b682c63
Why has been loaded, unable to see

**Comment by loppefaaret** [comment]
https://www.chiastatus.com/coins/0x72a64aa64d308cb14556727b2109fbf9a2a421d8144e4872f2e4c3ba2b682c63
it can be seen here

**Comment by BrandtH22** [comment]
@computer817 , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
